src/AppBundle/PullRequests/BodyParser.php

```diff
@@ -79,6 +81,14 @@ public function getCategory()
         return $this->extractWithRegex($regex);
     }
 
+    /**
+     * @return bool
+     */
+    public function isTestCategory()
+    {
+        return 1 === preg_match('/TE/', $this->getCategory());
```

Comment:
Use word boundaries to avoid matching "TEST" or "NOTES"
```suggestion
        return 1 === preg_match('/\\bTE\\b/', $this->getCategory());
```